src/main/java/com/task/dao/impl/ReviewsDaoImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.task.dao.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import com.task.dao.ReviewsDao;
+import org.springframework.stereotype.Repository;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class ReviewsDaoImpl implements ReviewsDao {

```

Comment:
```suggestion
public class ReviewDaoImpl implements ReviewDao {

```
class naming should be in a single [](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names)